rules/consistent-function-scoping.js

```diff
@@ -60,6 +60,13 @@ function checkReferences(scope, parent, scopeManager) {
 		);
 }
 
+const isReactHooks = scope =>
```

Comment:
```suggestion
const isReactHook = scope =>
```

It's only a single hook.